### ISSUE TITLE
The current module does not support Jsonpath expression like: $..book[?(@.price<10)]

### ISSUE THREAD

**Issue by jenliusa** [kennknowles/python-jsonpath-rw#34]
I get the following error message:
jsonpath_rw.lexer.JsonPathLexerError: Error

**Comment by ClericPy** [comment]
yup, I also want to know if this support '?(<expression>)' as https://github.com/jayway/JsonPath

**Comment by ClericPy** [comment]
does this https://pypi.python.org/pypi/jsonpath-rw-ext/0.1.9 fit...

**Comment by jenliusa** [comment]
Yes, it seems fit. I will take a deep look.
Thanks!!!
